src/Servers/Kestrel/Kestrel.slnf

```diff
@@ -2,11 +2,17 @@
   "solution": {
     "path": "..\\..\\..\\AspNetCore.sln",
     "projects": [
+      "src\\DataProtection\\Abstractions\\src\\Microsoft.AspNetCore.DataProtection.Abstractions.csproj",
```

Comment:
Are these changes required/intentional?

Reply:
They're required to build the Kestrel solution. The recently added web transport sample project uses WebApplication and references the `Microsoft.AspNetCore` project. It requires these dependencies.